### DISCUSSION TITLE
Can I use Vidstack with an older version of Node?

### DISCUSSION THREAD

**Discussion by martin-bachmann** [vidstack/player#1029] [Q&A] [answered]
Hi there,

I'm exploring the use of Vidstack to incorporate Vimeo videos as a background for a React project. Presently, our environment operates on Node version 14.19.0, whereas Vidstack requires version 16 or higher. Is there a possibility of utilizing Vidstack with Node 14.19.0, perhaps through an older version of the library?

Appreciate your attention to this matter!

Best regards,
Martin

**Comment by mihar-22** [comment, marked as answer]
Hey Martin! 

Node 14 has reached end-of-life (EOL) and most platforms have dropped it by now. It's no longer supported so I strongly recommend upgrading when you can. Unfortunately, we don't have an older version that's Node 14.x compatible. 

If for some reason you can't upgrade just yet, you can try installing whilst ignoring engines. Each package manager (NPM/Yarn/PNPM) has a possibly different CLI arg (e.g., `--ignore-engines`) so check out their docs on how to do so.

**Reply by martin-bachmann** [reply]
Thanks!
